pkg/util/mount/nsenter_mount.go

```diff
@@ -130,9 +133,19 @@ func (n *NsenterMounter) makeNsenterArgs(source, target, fstype string, options
 	nsenterArgs := []string{
 		"--mount=/rootfs/proc/1/ns/mnt",
 		"--",
-		n.absHostPath("mount"),
 	}
 
+	// fall back to simple mount when the host has no systemd
+	if systemdRunPath, hasSystemd := n.paths["systemd-run"]; hasSystemd {
+		descriptionArg := fmt.Sprintf("--description=Kubernetes transient mount for %s", target)
+		// Run with --scope to wait until /bin/mount finishes so we can grab
+		// its stdout/stderr. --scope also prevents systemd from killing fuse
+		// daemons when /bin/mount finishes successfully.
+		nsenterArgs = append(nsenterArgs, systemdRunPath, descriptionArg, "--scope", "--")
+	}
+
+	nsenterArgs = append(nsenterArgs, n.absHostPath("mount"))
```

Comment:
so if host has systemd, should we still append the n.absHostPath("mount") to the nsenterArgs?